### DISCUSSION TITLE
Custom structs and vector of custom structs modifiable from editor

### DISCUSSION THREAD

**Discussion by RicardRC** [GodotECS/godex#236] [Ideas & Proposals]
I'd need it for shared resources, but maybe it's useful for all kinds of resource.

**Comment by AndreaCatania** [comment]
This proposal is being implemented in the PR: https://github.com/GodotECS/godex/pull/237

**Reply by AndreaCatania** [reply]
and https://github.com/GodotECS/godex/pull/261
